test/unit/attributes.js

```diff
@@ -451,7 +451,7 @@ QUnit.test( "attr(String, Object)", function( assert ) {
 
 	$radio = jQuery( "<input>", {
 		"value": "sup",
-		"type": "radio"
+		"TYPE": "radio"
```

Comment:
Do we need that since the `value` case above tests all-lowercase? Or is there some specific concern about `type`?